cmd/dockerd/docker.go

```diff
@@ -30,6 +37,15 @@ func newDaemonCommand() (*cobra.Command, error) {
 		SilenceErrors: true,
 		Args:          cli.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if printDefaultSeccompProfile && printDefaultApparmorProfile {
+				return errors.New("conflicting flag: --print-default-seccomp-profile and --print-default-apparmor-profile")
+			}
+			if printDefaultSeccompProfile {
+				return daemon.PrintDefaultSeccompProfile(cmd.OutOrStdout())
+			}
+			if printDefaultApparmorProfile {
+				return daemon.PrintDefaultAppArmorProfile(cmd.OutOrStdout())
+			}
```

Comment:
Given that these cannot be combined, should we have a single flag that takes an option? Something like;

```
--print-default-profile=<seccomp|apparmor>
```

or (removing `-default` from the flag name)

```
--print-profile=<seccomp|apparmor>
```

Reply:
Could also be the reverse, and make it `--print-defaults` (so that we could, e.g. use it to print other defaults that are configured, such as `dns`, `ulimit`, etc)